cmd/apiserver/main.go

```diff
@@ -187,13 +139,12 @@ func (b *backendProvider) LoadBalancers() ([]*repository.LoadBalancer, error) {
 func main() {
 	log := logger.New()
 
-	options, err := gatherOptions()
```

Comment:
Why do we need to remove this function? It enables unit testing: performing the options collection directly in main results in code that is more challenging to test.